docker/deb-ci/Dockerfile_template

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:CYCLUS_UBUNTU_MAJOR_VERSION.04
+
+# Because we used specific Ubuntu version can't use cyclus-deps docker container
+RUN apt-get -y --force-yes update
+
+RUN apt-get install -y --force-yes \
+    cmake \
+    make \
+    libboost-all-dev \
+    libxml2-dev \
+    libxml++2.6-dev \
+    libsqlite3-dev \
+    libhdf5-serial-dev \
+    libbz2-dev \
+    coinor-libcbc-dev \
+    coinor-libcoinutils-dev \
+    coinor-libosi-dev \
+    coinor-libclp-dev \
+    coinor-libcgl-dev \
+    libblas-dev \
+    liblapack-dev \
+    g++ \
+    libgoogle-perftools-dev \
+    git \
+    python \
+    python-tables \
+    python-numpy \
+    python-nose
+
+COPY . /cyclus 
+WORKDIR cyclus
+RUN mkdir -p deb
+WORKDIR /cyclus/deb
+RUN cmake -DCMAKE_INSTALL_PREFIX=/usr -DCPACK_HASH_NAME=CYCLUS_REPO_HASH -DCPACK_SYSTEM_NAME=ubuntu.CYCLUS_UBUNTU_MAJOR_VERSION.04 ..
```

Comment:
Despite previously defining  -DCMAKE_INSTALL_PREFIX=/usr/local, cyclus file are installed (when using dpkg -i) in /usr directly...
So cyclus is looking for /usr/local... but don't find its stuff...

when  switch to -DCMAKE_INSTALL_PREFIX=/usr it is working.
So we can either using /usr as install prefix or try to understand why dpkg install cyclus in /usr despite the cpack/cmake specification...